LongestPalindrome.py

```diff
@@ -0,0 +1,25 @@
+#Time Complexity : O(n))
+#Space Complexity : O(n) for dictionary, although we can use a hashset for O(1) space complexity, not sure how to do it in python
+#Did this code successfully run on Leetcode : yes
+#Any problem you faced while coding this : I am not sure why but leet code said I am only in the 30th percentile for time complexity and
```

Comment:
Sometimes even the most efficient solution doesn't show good stats, it's okay not to rely on the leetcode